src/main/scala/viper/silver/ast/utility/BackendTypeFactories.scala

```diff
@@ -15,8 +15,10 @@ case class BVFactory(size: Int) {
   def or(name: String) = BackendFunc(name, "bvor", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
   def add(name: String) = BackendFunc(name, "bvadd", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
   def mul(name: String) = BackendFunc(name, "bvmul", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
+  def udiv(name: String) = BackendFunc(name, "bvudiv", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
+  def urem(name: String) = BackendFunc(name, "bvurem", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
   def shl(name: String) = BackendFunc(name, "bvshl", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
-  def shr(name: String) = BackendFunc(name, "bvshr", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
+  def lshr(name: String) = BackendFunc(name, "bvlshr", typ, Seq(LocalVarDecl("x", typ)(), LocalVarDecl("y", typ)()))
```

Comment:
Maybe `bvashr` as well if it's ever used in the future?

Reply:
Does it exist? I did not see it in the documentation.

Reply:
Sure.
```
(declare-const x (_ BitVec 2))
(assert (= x (bvashr #b10 #b01)))
(check-sat)
(get-model)

sat
(model 
  (define-fun x () (_ BitVec 2)
    #b11)
)
```

Reply:
Thanks! Added.